### PR TITLE
add info/about.json

### PR DESCRIPTION
**Pull request by ilanschnell**
In this PR, we add `info/about.json` to the conda package, which contains the following keys:
```
home, dev_url, doc_url, license_url, license, summary, description, license_family
```
There are simply copied from the `about` section in `meta.yaml` (if they exist).
This addresses https://github.com/conda/conda-build/issues/939

### CODE REVIEW

**Comment by msarahan** [comment]
I'm not sure about your simplified file writing.  I think it should work - and the Travis tests seem happy enough, but do you know why that was there in the first place?

**Comment by ilanschnell** [comment]
I have no idea why the file writing was done so complicated in the first place.

File: conda_build/build.py [hunk 1]
```diff
@@ -218,13 +217,20 @@ def create_info_files(m, files, include_recipe=True):
             info_index['depends'] = [' '.join(dist.rsplit('-', 2))
                                      for dist in dists]
 
-    # Deal with Python 2 and 3's different json module type reqs
-    mode_dict = {'mode': 'w', 'encoding': 'utf-8'} if PY3 else {'mode': 'wb'}
```
Comment (groutr) [inline conda_build/build.py]: I think it might be important that we keep the `encoding: uft-8` part.

**Comment by ilanschnell** [comment]
What is the status on this PR?

**Comment by msarahan** [comment]
I am uneasy about your simplification of file writing.  Please remove that from your PR, and I'm happy to merge your info.json file addition.

**Comment by ilanschnell** [comment]
@msarahan, here you go

**Comment by msarahan** [comment]
thanks

**Pull request merged**